cabal-install/Distribution/Client/Types.hs

```diff
@@ -31,11 +32,13 @@ import Distribution.Version
 
 import Distribution.Solver.Types.PackageIndex
          ( PackageIndex )
+import qualified Distribution.Solver.Types.ComponentDeps as CD
 import Distribution.Solver.Types.ComponentDeps
          ( ComponentDeps )
 import Distribution.Solver.Types.OptionalStanza
 import Distribution.Solver.Types.PackageFixedDeps
 import Distribution.Solver.Types.SourcePackage
+import Distribution.Compat.Graph (IsNode(..))
```

Comment:
BTW, could we at some rename this module. It really doesn't deserve to live in Compat, it's a module adding real functionality not just papering over version compatibility issues.